tests/robottelo/test_decorators.py

```diff
@@ -36,20 +38,49 @@ def test_not_smoke(self):
 class BzBugIsOpenTestCase(TestCase):
     """Tests for :func:`robottelo.decorators.bz_bug_is_open`."""
     # (protected-access) pylint:disable=W0212
-    def setUp(self):  # noqa pylint:disable=C0103
```

Comment:
Thanks for dropping the `noqa`. It appears that flake8 has been updated to ignore the `setUp` and `setUpClass` names.